Islandora/islandora
# Delete islandora_text_extraction.install

**Github Issue**: Discovered while reviewing https://github.com/Islandora/documentation/pull/1332

# What does this Pull Request do?

Removes an unneccessary install hook.

# What's new?
No hook in the text extraction submodule, since the extension it adds (.txt) is on File media entities by default.

# How should this be tested?

When going through the final Drupal configuration steps in https://github.com/Islandora/documentation/pull/1332, you should no longer have to run the `drush en` command twice.  See https://github.com/Islandora/documentation/pull/1332/files#r355680955

# Interested parties
@Islandora/8-x-committers
